third_party/move/move-binary-format/src/compatibility.rs

```diff
@@ -72,47 +73,61 @@ impl Compatibility {
     }
 
     /// Check compatibility for `new_module` relative to old module `old_module`.
-    pub fn check(&self, old_module: &Module, new_module: &Module) -> PartialVMResult<()> {
-        let mut struct_and_pub_function_linking = true;
-        let mut struct_layout = true;
-        let mut friend_linking = true;
+    pub fn check(
+        &self,
+        old_module: &CompiledModule,
+        new_module: &CompiledModule,
+    ) -> PartialVMResult<()> {
+        let mut errors = vec![];
+        //let mut struct_layout_errors = vec![];
```

Comment:
Can you remove these comments in the next enum PR?

Reply:
Oops. Done right away.